integration/03_cluster_id.go

```diff
@@ -0,0 +1,36 @@
+// +build k8srequired
+
+package integration
+
+import (
+	"fmt"
+	"os"
+)
+
+const (
+	// EnvVarClusterID is the process environment variable representing the
+	// CLUSTER_NAME env var.
+	//
+	// TODO rename to CLUSTER_ID. Note this also had to be changed in the
+	// framework package of e2e-harness.
+	EnvVarClusterID = "CLUSTER_NAME"
+)
+
+// init sets the cluster ID of the current integration test to the process
+// environment at runtime. init of cluster_id.go is called after init of
+// circle_sha.go. Execution of multiple init functions within a package is
+// guaranteed to happen in lexicographical order.
+//
+//     To ensure reproducible initialization behavior, build systems are
+//     encouraged to present multiple files belonging to the same package in
+//     lexical file name order to a compiler.
+//
+// NOTE Implications of changing file names means breaking the initialization
+// behaviour.
+func init() {
+	os.Setenv(EnvVarClusterID, ClusterID())
+}
+
+func ClusterID() string {
+	return fmt.Sprintf("ci-awsop-%s-%s", TestedVersion(), CircleSHA()[0:5])
+}
```

Comment:
I think managing this in the code is cooler than in the circle configs. That way we can also have a common package supporting basic integration test functionalities. Maybe we even want to move this to e2e-harness at some point.

Reply:
The problem with this is that we can't set a cluster id for local runs in an environment variable. Also, we would need to export `CIRCLE_SHA` locally for getting a cluster id, and this environment variable is also used for determining the chart version to use.

Reply:
What about only writing the `CLUSTER_NAME` env var in case it is not set. Then you can predefine it and it does not get overwritten while we keep the support of the programming language here, which I want to keep.

Reply:
Yes, that makes sense. I think it would be good to always take into account already set env vars, and only use logic when needed.

Reply:
Done.